bin/powershell/check-windows-directory.ps1

```diff
@@ -0,0 +1,33 @@
+<#
+.SYNOPSIS 
+    Checks if  directory exist
+.DESCRIPTION
+    Checks if directory exist
+.Notes
+    FileName    : check-windows-directory.ps1
+    Author      : Patrice White - patrice.white@ge.com
+.LINK 
+    https://github.com/sensu-plugins/sensu-plugins-windows
+.PARAMETER LogName 
+    Required. The name of the directory.
+    Example -Dir C:\Users\dir
+.EXAMPLE
+    powershell.exe -file check-windows-directory.ps1 -Dir C:\Users\dir
+#>
+
+[CmdletBinding()]
+Param(
+  [Parameter(Mandatory=$True)]
+  [string]$Dir
+)
+
+$ThisDir = Test-Path -Path $Dir

```

Comment:
would this work if I passed a directory on another drive? I recall there being weird things about needing to switch drives before accessing directories from them but that could be something that no longer applies (I clearly have not used windows in a long time).

Reply:
@majormoses Yes as long as you have the path to the drive it will work.